charts/openshift-metering/values.yaml

```diff
@@ -276,6 +276,12 @@ reporting-operator:
         createSecret: false
         secretName: ""
 
+      azure:
```

Comment:
At this moment, I don't believe reporting-operator needs this information. These aren't used.